server/channels/api4/group.go

```diff
@@ -301,6 +302,14 @@ func patchGroup(c *Context, w http.ResponseWriter, r *http.Request) {
 	auditRec.AddEventResultState(group)
 	auditRec.AddEventObjectType("group")
 
+	c.App.Srv().GetTelemetryService().SendTelemetryForFeature(
+		telemetry.TrackGroupsFeature,
+		"modify_group_edit_details",
```

Comment:
Perhaps move event name to constants.

Reply:
Thinking out loud. There's a few `modify_group_<action>_<item>` or `modify_group_<verb>_<noun>`. If I understand correctly, the `modify_group` prefix is used in order to avoid  naming conflict with other events when the `<action>_<item>` part seems likely to be reused. This is a great idea, and it greatly helps when processing the events downstream. 

@catalintomai there's two possible options here:
- Adopt this pattern. Perhaps we can make the existence of the prefix explicit, by requiring prefix to be separated with double underscore from the event name (i.e. `modify_group__edit_details`).
- Avoid the event and use more explicit naming. I.e. rather than `modify_group_edit_details` use `edit_user_group_details`. This also lowers the chance of naming conflict.

What do you think?